metals/src/main/scala/scala/meta/internal/metals/InteractiveSemanticdbs.scala

```diff
@@ -178,8 +180,15 @@ final class InteractiveSemanticdbs(
     // NOTE(olafur): it's unfortunate that we block on `semanticdbTextDocument`
     // here but to avoid it we would need to refactor the `Semanticdbs` trait,
     // which requires more effort than it's worth.
+    val params = new CompilerVirtualFileParams(
+      source.toURI,
+      modifiedText,
+      token = EmptyCancelToken,
+      outlineFiles =
+        compilers().outlineFilesProvider.getOutlineFiles(optBuildTarget),
```

Comment:
We should probably move this inside compilers itself, so that outlineFilesProvider is not touched outside of compilers.